packages/codemirror-extension/schema/plugin.json

```diff
@@ -28,6 +28,49 @@
       "selector": ".cm-content"
     }
   ],
+  "jupyter.lab.menus": {
+    "main": [
+      {
+        "id": "jp-mainmenu-view",
```

Comment:
I think view menu is fine. The alternative are "Edit" or "Settings". RStudio uses "Edit"